### PR TITLE
release-22.2: ui: added common id URL parameter constant for jobs, schedules, insights

### PR DESCRIPTION
**Pull request by ericharmeling**
Backport 1/1 commits from #92932.

/cc @cockroachdb/release

---

This commit adds a common URL parameter constant and parameter selector for the jobs, schedules, and insights details pages.

Part of https://github.com/cockroachdb/cockroach/issues/87693.

Background thread: https://cockroachlabs.slack.com/archives/G01Q9D01NTU/p1670001751479989

Release note: None

Release justification: bug fix

### CODE REVIEW

**Review by maryliag — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/94944#-:-NLNTTS7B8wR8sMvkHnD:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained

**Comment by ericharmeling** [comment]
Note before merging: 

This PR reverts portions of https://github.com/cockroachdb/cockroach/pull/92276 that break the current CC console for 22.2 `cluster-ui` builds (specifically the ID URL parameter parsing from the sidenav for insights e.g., https://github.com/cockroachdb/cockroach/pull/92276/files#diff-76f1a8e64f572b594f073139a74e6d963eb975f080e4572783a1dd853cbb5b22L101). 

This PR does not fix https://github.com/cockroachdb/cockroach/issues/94380 in 22.2, which I believe was also introduced in https://github.com/cockroachdb/cockroach/pull/92276. That issue is fixed in 23.1 with https://github.com/cockroachdb/cockroach/pull/92285, which will not be backported to 22.2. So, I will follow up this PR with another one resolving https://github.com/cockroachdb/cockroach/issues/94380 in 22.2.

For context, the 22.2 CC console currently uses `cluster_ui@22.2.0`, which was published before https://github.com/cockroachdb/cockroach/pull/92276 was merged.

@maryliag We should hold off on publishing a `cluster_ui@22.2.1` until we have all bugs fixed.

**Pull request merged**
